codes/quantum/properties/approximate_oaecc.yml

```diff
@@ -0,0 +1,47 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id : approximate_oaecc
+
+name: 'Approximate operator-algebra error-correcting code'
```

Comment:
Since you're going down this route, check this paper to see if you can include it: https://arxiv.org/abs/1806.10324